app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,8 @@
 <resources>
     <string name="app_name">EasySurvey</string>
+
+
+    <string name="take_survey">Take Survey></string>
+    <string name="create_survey">Create Survey</string>
```

Comment:
Added various strings within the strings.xml file for potential future use later on in the project

Reply:
This looks like an incomplete feature.
Are you using these anywhere?  I don't see them used.
If not in use, remove them.  Keep the codebase as small as possible to do the job you want to do.  Unused code is clutter, and bad documentation, which increases technical debt.
Only complete features, which meet the team's Definition of Done, should be merged to Main/Master.